fdbclient/MultiVersionTransaction.actor.cpp

```diff
@@ -1891,13 +1891,25 @@ void MultiVersionApi::loadEnvironmentVariableNetworkOptions() {
 		return;
 	}
 
+	FDBOptionParamType curParamType;
+	int64_t intParamVal;
 	for (auto option : FDBNetworkOptions::optionInfo) {
 		if (!option.second.hidden) {
 			std::string valueStr;
 			try {
 				if (platform::getEnvironmentVar(("FDB_NETWORK_OPTION_" + option.second.name).c_str(), valueStr)) {
+					curParamType = option.second.paramType;
 					for (auto value : parseOptionValues(valueStr)) {
-						Standalone<StringRef> currentValue = StringRef(value);
+						Standalone<StringRef> currentValue;
+						if (curParamType == Int) {
+							char* end;
+							intParamVal = strtoull(value.c_str(),&end,10);
+							ASSERT(!(*end) && intParamVal);
+							currentValue = StringRef((uint8_t*)(&intParamVal),8);
```

Comment:
This works as written, though we could instead use a C++ style cast here for a little more clarity:

```suggestion
							currentValue = StringRef(reinterpret_cast<uint8_t*>(&intParamVal), 8);
```